package.json

```diff
@@ -17,6 +17,7 @@
 		"build:docs": "nx build storybook --output-dir ../dist/",
 		"build:preview": "nx build storybook",
 		"builder": "nx run-many --target build report --projects",
+		"bundle": "nx build bundle",
```

Comment:
The clean command didn't actually remove tools/bundle/dist, here's the output:
<img width="1024" alt="image" src="https://github.com/user-attachments/assets/8281f7b1-2715-4131-ac79-db6380cebfe7" />

Reply:
It won't if it's cached but if I set clean commands to ignore the cache by default, they will run every build command every time, even if it should be cached. I could update the package.json clean command to skip cache though.

Reply:
I'm not sure it's a problem, just reporting since you had noted the command. Practically speaking, I'm not sure there's a reason to do this command adhoc so maybe it's not an issue worth spending time on?